lms/djangoapps/discussion_api/api.py

```diff
@@ -72,6 +74,20 @@ def _get_thread_and_context(request, thread_id, parent_id=None, retrieve_kwargs=
         raise Http404
 
 
+def _is_user_author_or_privileged(cc_thread, context):
+    """
+    Check if the user is the author of a thread or a privileged user. Such users
+    may update or delete said thread.
```

Comment:
as this is a broadly reusable helper, i'd remove the second sentence of the docstring.   (i also think "Returns: boolean" would suffice)